docs/api/history.md

```diff
@@ -190,3 +212,133 @@ repo.history.remove(two)
 
 repo.history.toArray() // [root, one, three]
 ```
+
+## Events
+
+History emits events any time something of interest happens. This is
+how Microcosm knows to update the state to accurately reflect what's
+going on given a sequence of actions and their varying statuses.
+
+You can manage event listeners with the following methods.
+
+### `on(event, callback)`
+
+Adds an event listener to a Microcosm History instance.
+```javascript
+const repo = new Microcosm()
+const history = repo.history
+
+history.on('append', callback)
+```
+
+### `off(event, callback)`
+
+Remove an event listener.
```

Comment:
To stay consistent with description of `on`, maybe change this to "Removes an event listener".